### PR TITLE
feat(blog): alexa custom TTS summary post

### PR DESCRIPTION
**Pull request by space-pope**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

A stub post that links both to the Voice Tech Podcast article on custom Alexa TTS and straight to the GitHub repo. Open to taking out the latter link if we want; figured I'd give as many paths as possible, but maybe it makes sense to just promote the other piece.

### CODE REVIEW

File: content/blog/how-to-change-alexas-voice/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: How to Change Alexa's Voice
+date: '2021-06-07'
+description: Use Spokestack's TTS service to avoid your skill sounding like all the others.
+author: josh
+tags: 3rd Party Integrations, Tutorial
+draft: false
+hero: ../../assets/blog/how-to-change-alexas-voice.png
```
Comment (space-pope) [inline content/blog/how-to-change-alexas-voice/index.md:8]: @timmywil Do I also want `seoImage` here, or is `hero` sufficient?

File: content/blog/how-to-change-alexas-voice/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+![How to Change Alexa's Voice](../../assets/blog/how-to-change-alexas-voice.png)
+
+At Spokestack, one of our core goals is to make it possible for every mobile app or voice skill to have its own unique audible voice. Amazon and Google make it easy to use their text-to-speech services, but then your skill (/action) ends up sounding like all the rest. If you've ever wondered how to change Alexa's voice, we have an [article on Voice Tech Podcast](https://voicetechpodcast.com/articles/development/how-to-change-alexas-voice/) just for you.
+
+It'll give you some background on what's required to actually replace the system voice and walk you step by step through the use of [some sample code](https://github.com/spokestack/alexa-custom-tts) to try it out for yourself. You can use a free Spokestack account to try everything out, and if you agree that your skill should stand out from the crowd, you can upgrade to [the Maker tier](/pricing#maker) and create a completely new voice that only you can use.
```
Comment (noelweichbrodt) [inline content/blog/how-to-change-alexas-voice/index.md]:
```suggestion
It'll give you some background on what's required to actually replace the system voice and walk you step by step through the use of [a low-code example in Python](https://github.com/spokestack/alexa-custom-tts) to set up a skill of your own. You can use a free Spokestack account to try everything out, and if you agree that your skill should stand out from the crowd, you can upgrade to [the Maker tier](/pricing#maker) and create a completely new voice that only you can use.
```

Comment (noelweichbrodt) [reply content/blog/how-to-change-alexas-voice/index.md]: just wanted to work `low-code` in there

**Review by noelweichbrodt — APPROVED** [review]

**Comment by timmywil** [comment]
@space-pope I think you must have read my comment about how to use fix() and feat() commit tags. Thanks! In addition to those, it is also valid to use `fixup!` or `squash!` in PRs to designate commits to be squashed.

**Comment by space-pope** [comment]
Yeah; I've been trying to follow that convention for a bit now, but probably messed up occasionally. Didn't know the semantics of the other two, though, so thanks for the tip.

**Pull request merged**